addon/components/au-alert.gts

```diff
@@ -2,12 +2,12 @@ import { on } from '@ember/modifier';
 import { action } from '@ember/object';
 import Component from '@glimmer/component';
 import { tracked } from '@glimmer/tracking';
-import AuIcon from './au-icon';
+import AuIcon, { type AuIconSignature } from './au-icon';
 
 export interface AuAlertSignature {
   Args: {
     closable?: boolean;
-    icon?: string;
+    icon?: AuIconSignature['Args']['icon'];
```

Comment:
I think we need similar changes in a couple of other components that also accept configurable icons:

- au-accordion
- au-dropdown
- au-link-external

Reply:
Huh, I thought I'd searched for all of these, I'll change those and do another search through for any I missed.

Reply:
ah, au-dropdown is still a .gjs file, so ignore that one. Maybe I converted the others after you created your PR, not sure 😄.